### ISSUE TITLE
Assignment error when executing again on the same data/

### ISSUE THREAD

**Issue by Akado2009** [griffithlab/pVAC-Seq#220]
There appears to be an error in your pipeline.py when I try to execute again pvacseq on the same data:
```
$ pvacseq run ./input_data/input.vcf Test HLA-G*01:09,HLA-E*01:01 NetMHC PickPocket . -e 9,10
Executing MHC Class I predictions
Converting VCF to TSV
Completed
Splitting TSV into smaller chunks
Splitting TSV into smaller chunks - Entries 1-24
Split TSV file for Entries 1-24 already exists. Skipping.
Traceback (most recent call last):
  File "/usr/local/bin/pvacseq", line 11, in <module>
    load_entry_point('pvacseq==4.0.1', 'console_scripts', 'pvacseq')()
  File "/home/akado2009/.local/lib/python3.5/site-packages/pvacseq/pvacseq.py", line 77, in main
    args[0].func.main(args[1])
  File "/home/akado2009/.local/lib/python3.5/site-packages/pvacseq/lib/main.py", line 252, in main
    pipeline.execute()
  File "/home/akado2009/.local/lib/python3.5/site-packages/pvacseq/lib/pipeline.py", line 238, in execute
    chunks = self.split_tsv_file(total_row_count)
  File "/home/akado2009/.local/lib/python3.5/site-packages/pvacseq/lib/pipeline.py", line 134, in split_tsv_file
    split_tsv_file.close()
UnboundLocalError: local variable 'split_tsv_file' referenced before assignment
```
An errors appears to be  here:
```
f os.path.exists(split_tsv_file_path):
                print("Split TSV file for Entries %d-%d already exists. Skipping." % (split_start, split_end))
                skip = 1
            else:
                split_tsv_file      = open(split_tsv_file_path, 'w')
                split_tsv_writer    = csv.DictWriter(split_tsv_file, delimiter='\t', fieldnames = reader.fieldnames)
                split_tsv_writer.writeheader()
                skip = 0
            for row in reader:
                if skip == 0:
                    split_tsv_writer.writerow(row)
                if row_count % tsv_size == 0:
                    if skip == 0:
                        split_tsv_file.close()
                    split_start = row_count + 1
                    split_end   = split_start + tsv_size - 1
                    if split_end > total_row_count:
                        split_end = total_row_count
                    print("Splitting TSV into smaller chunks - Entries %d-%d" % (split_start, split_end))
                    split_tsv_file_path = "%s_%d-%d" % (self.tsv_file_path(), split_start, split_end)
                    chunks.append([split_start, split_end])
                    if os.path.exists(split_tsv_file_path):
                        print("Split TSV file for Entries %d-%d already exists. Skipping." % (split_start, split_end))
                        skip = 1
                    else:
                        split_tsv_file      = open(split_tsv_file_path, 'w')
                        split_tsv_writer    = csv.DictWriter(split_tsv_file, delimiter='\t', fieldnames = reader.fieldnames)
                        split_tsv_writer.writeheader()
                        skip = 0
                row_count += 1
split_tsv_file.close()
```
You are trying to close file by name without opening any file since if I've already executed it on my data, the next time I try it it will have skip flag equal to 1 and I will never have split_tsv_file as a variable.
Thx in advance!

**Comment by susannasiebert** [comment]
This error was addressed in pull request #211 and will be available in the next release (probably later today).

**Comment by susannasiebert** [comment]
This bug has been fixed in version 4.0.2.
